cime/config/e3sm/machines/Depends.cray.cmake

```diff
@@ -1 +1 @@
-list(NOOPT_FILES APPEND cice/src/mpi/ice_boundary.F90 cice/src/serial/ice_boundary.F90)
+list(APPEND NOOPT_FILES cice/src/mpi/ice_boundary.F90 cice/src/serial/ice_boundary.F90 cice/src/source/ice_domain.F90 cam/src/physics/cam/gw_common.F90)
```

Comment:
Having `ice_domain.F90` and `gw_common.F90` at `-O2` or `-O1` was producing non-BFB results in ERS tests of the `e3sm_integration` test suite. Compiling them with `-O0` made the tests BFB.